compiler/ballerina-lang/src/main/java/io/ballerina/projects/JBallerinaBackend.java

```diff
@@ -334,7 +334,8 @@ public void performCodeGen(ModuleContext moduleContext, CompilationCache compila
         if (bLangPackage.getErrorCount() > 0) {
             return;
         }
-        CompiledJarFile compiledJarFile = jvmCodeGenerator.generate(bLangPackage);
+        boolean remoteManagement = moduleContext.project().buildOptions().compilationOptions().remoteManagement();
```

Comment:
Any suggestions for the boolean variable name? `isRemoteManagementEnabled` or `isRemoteMgtEnabled`? Dropping `management` part (`isRemoteEnabled`) may bring confusion and reduce the readability since we have other usages of the word `remote`.